### DISCUSSION TITLE
Kanban board

### DISCUSSION THREAD

**Discussion by koppanyinorbert** [usememos/memos#1593] [Ideas]
I tried a lot of apps for a Kanban board but I can't find a similar lightweight app like this. 
Is it possible to add a Kanban board?
Thanks! 🙏

**Comment by lephtHanded** [comment]
@koppanyinorbert  - have you looked at [tasks.md](https://github.com/BaldissaraMatheus/Tasks.md)

**Reply by koppanyinorbert** [reply]
Oh my god, NO! 
Thank you very much, I will try it !
